pyanaconda/ui/gui/spokes/user.py

```diff
@@ -156,7 +156,12 @@ def apply(self):
         else:
             self._user.gid = None
 
-        self._user.groups = [g.strip() for g in self._tGroups.get_text().split(",")]
+        groups_string = self._tGroups.get_text()
+        # ''.split(',') returns [''] instead of [], which is not what we want
+        if groups_string:
+            self._user.groups = [g.strip() for g in groups_string.split(",")]
+        else:
+            self._user.groups = []
```

Comment:
Wouldn't adding ``if g`` (or better ``if g.strip()``) to the end of the list comprehension do the same (plus check for empty strings as groups)?